packages/shared/components/molecules/NftMedia.svelte

```diff
@@ -27,6 +27,7 @@
 {:else}
     <MediaDisplay
         filePath={nft.filePath}
+        url={nft.downloadUrl}
         isLoaded={nft.downloadMetadata.isLoaded}
         expectedType={nft.parsedMetadata.type}
```

Comment:
Might be worth passing the entire `nft` object as a `media` object and make `MediaDisplay` smarter to decide what to do with it. Could be part of another PR.

Reply:
the reasoning behind it is that we could potentially use the `MediaDisplay` independent of nfts. So i designed the MediaDisplay NFT-agnostic